modules/olm-installing-opm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-managing-custom-catalogs.adoc
+
+[id="olm-installing-opm_{context}"]
+= Installing `opm`
+
+To install the `opm` CLI tool on your workstation:
```

Comment:
Odd to have this intro sentence fragmented here. I'd write it out. _You can install the `opm` CLI tool on your workstation._